kolena/_utils/consts.py

```diff
@@ -21,3 +21,9 @@ class BatchSize(int, Enum):
 
     LOAD_RECORDS = UPLOAD_RECORDS
     LOAD_SAMPLES = 1_000_000
+
+
+class FieldNames(str, Enum):
```

Comment:
Usually a name like this would be singular — see `BatchSize` above

Reply:
Will updated

Reply:
Will update

Reply:
Done